openmeter/meter/httphandler/query.go

```diff
@@ -205,3 +205,74 @@ func (h *handler) QueryMeterPost() QueryMeterPostHandler {
 		)...,
 	)
 }
+
+type (
+	ListGroupByValuesResponse = *[]string
+	ListGroupByValuesHandler  httptransport.HandlerWithArgs[ListGroupByValuesRequest, ListGroupByValuesResponse, ListGroupByValuesParams]
+)
+
+type ListGroupByValuesParams struct {
+	IdOrSlug   string
+	GroupByKey string
+	From       *time.Time
+	To         *time.Time
+	Search     *string
+}
+
+type ListGroupByValuesRequest struct {
+	namespace  string
+	idOrSlug   string
+	groupByKey string
+	from       *time.Time
+	to         *time.Time
+	search     *string
+}
+
+// ListGroupByValues returns a handler for list group by values.
+func (h *handler) ListGroupByValues() ListGroupByValuesHandler {
+	return httptransport.NewHandlerWithArgs(
+		func(ctx context.Context, r *http.Request, params ListGroupByValuesParams) (ListGroupByValuesRequest, error) {
+			ns, err := h.resolveNamespace(ctx)
+			if err != nil {
+				return ListGroupByValuesRequest{}, err
+			}
+
+			return ListGroupByValuesRequest{
+				namespace:  ns,
+				idOrSlug:   params.IdOrSlug,
+				groupByKey: params.GroupByKey,
+				from:       params.From,
```

Comment:
let's add a validation that from-to duration cannot be more than 90 days or that from must be younger than 90 days so we don't scan more than 3 partitions